### DISCUSSION TITLE
Getting Started Documentation

### DISCUSSION THREAD

**Discussion by chrisativonne** [11ty/eleventy#2940] [Uncategorized]
We have used 11ty for the past few years to quickly reproduce backup (simplified) copies of our existing websites.
It has been over a year since I have looked at it and I'm digging back into how to set up 11ty again to manage a library of html email templates.

There used to be a really good walk through guide on the use of templates and layouts, but I can no longer find it.

Why is 11ty to temperamental? One file in the wrong place and the entire project won't update. There must be a better way to isolate non-compliant files/errors so that the entire project doesn't get taken down from watch.

**Comment by bobmonsour** [comment]
There are a lot of getting started blog posts here: https://11tybundle.dev/categories/getting-started/

Perhaps it's somewhere among these.
